src/Sarif.UnitTests/SarifTransformerTests.cs

```diff
@@ -168,5 +170,93 @@ public void SarifTransformerTests_ToCurrent_OneRunWithLogicalLocations()
             logicalLocations["collections"].Kind.Should().Be("namespace");
             logicalLocations["collections"].ParentKey.Should().BeNull();
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_OneRunWithFiles()
+        {
+            string content = "The quick brown fox jumps over the lazy dog";
+            byte[] bytes = Encoding.ASCII.GetBytes(content);
```

Comment:
The spec says to use UTF-8. It doesn't matter here, but your resident SARIF experts ;-) will trip over the use of `ASCII`.

Reply:
OK